File: Exercices/ExoJava/Exo3Java/Exo3.java
package Exercices.ExoJava.Exo3Java;


import java.util.Scanner;

public class Exo3 {
    public static boolean isEven(int number) {
        return number % 2 == 0;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);


        System.out.print("Entrez un nombre pour vérifier s'il est pair : ");
        int number = scanner.nextInt();

        if (isEven(number)) {
            System.out.println(number + " est un nombre pair.");
        } else {
            System.out.println(number + " est un nombre impair.");
        }

        scanner.close();
    }
}

File: Exercices/ExoC/Exo3C/Exo3.c
#include <stdio.h>

int is_even(int number) {
    return number % 2 == 0;
}

int main() {
    int number;


    printf("Entrez un nombre pour vérifier s'il est pair : ");
    scanf("%d", &number);

    if (is_even(number)) {
        printf("%d est un nombre pair.\n", number);
    } else {
        printf("%d est un nombre impair.\n", number);
    }

    return 0;
}
